### DISCUSSION TITLE
inflow module

### DISCUSSION THREAD

**Discussion by Chengwenju** [iiasa/CWatM#59] [Q&A]
Hello everyone:

I have a question about the inflow module. When we set two or more inflow files, the model does not run, but it can run when there is only one inflow file. Therefore, we believe that there is a problem with the format of the inflow files, but we have not been able to solve this problem so far. We hope someone can help us solve this problem. 

Thank you very much!

**Comment by dof1985** [comment]
Hello Chengwenju,

Please provide a reproducible example, e.g., what are the definition in your settings file, and how does the input files look like

**Comment by dxyu0101** [comment]
Hello, I encountered the same problem as you, when there are two input files, the model fails to run, would like to ask how you solve this problem

**Comment by FEER-0PS** [comment]
Hello，I'm also encountering format issues. Could you please share your successfully run inflow file format for reference?Thank you very much!

**Comment by mikhailsmilovic** [comment]
Guten Tag!
inflow = True

In_Dir = _folder_with_inflow_data_
We demonstrate this using a single tss file with several columns. 

InflowPoints = _list of coordinates of the inflow points_ 
example:
InflowPoints = 22.49 48.06 22.49 48.04 22.49 47.96 22.49 47.81 22.49 47.74

QInTS = _tss of inflow time series_ 
example:
QInTS = $(In_Dir)/inflow1990_2022.tss
[inflow1990_2022_.txt](https://github.com/user-attachments/files/25743840/inflow1990_2022_.txt)
